### PR TITLE
feat(DENG-582): commented out fxa_content_ views logic and raising error using them

### PR DESCRIPTION
**Pull request by kik-kik**
feat(DENG-582): commented out fxa_content_ views logic and raising error using them

The error contains a deprecation message along with additional information.

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_events/view.sql [hunk 1]
```diff
@@ -23,81 +23,85 @@
...
-  JSON_VALUE(user_properties, "$.sync_active_devices_week") AS sync_active_devices_week,
-  JSON_VALUE(user_properties, "$.sync_active_devices_month") AS sync_active_devices_month,
-  JSON_VALUE(event_properties, "$.email_sender") AS email_sender,
-  JSON_VALUE(event_properties, "$.email_service") AS email_service,
-  JSON_VALUE(event_properties, "$.email_template") AS email_template,
-  JSON_VALUE(event_properties, "$.email_version") AS email_version,
-FROM
-  unioned
+  ERROR(
+    'VIEW BEING DEPRECATION - This view will be completely deleted after 1st of February 2023, please use `fxa_all_events` with filter on `event_category` instead. See DENG-582 for more info.'
```
Comment (relud) [inline sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_events/view.sql:106]:
```suggestion
    'VIEW DEPRECATED - This view will be completely deleted after 1st of February 2023, please use `fxa_all_events` with filter on `event_category` instead. See DENG-582 for more info.'
```

File: sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_oauth_events/view.sql [hunk 2]
```diff
@@ -24,105 +24,109 @@
...
-  JSON_VALUE(user_properties, "$.sync_active_devices_week") AS sync_active_devices_week,
-  JSON_VALUE(user_properties, "$.sync_active_devices_month") AS sync_active_devices_month,
-  JSON_VALUE(event_properties, "$.email_sender") AS email_sender,
-  JSON_VALUE(event_properties, "$.email_service") AS email_service,
-  JSON_VALUE(event_properties, "$.email_template") AS email_template,
-  JSON_VALUE(event_properties, "$.email_version") AS email_version,
-FROM
-  unioned
+  ERROR(
+    'VIEW BEING DEPRECATION - This view will be completely deleted after 1st of February 2023, please use `fxa_all_events` with filter on `event_category` instead. See DENG-582 for more info.'
```
Comment (relud) [inline sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_oauth_events/view.sql:131]:
```suggestion
    'VIEW DEPRECATED - This view will be completely deleted after 1st of February 2023, please use `fxa_all_events` with filter on `event_category` instead. See DENG-582 for more info.'
```

File: sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_stdout_events/view.sql [hunk 3]
```diff
@@ -25,109 +25,113 @@ CREATE OR REPLACE VIEW
...
-  JSON_VALUE(event_properties, "$.plan_id") AS plan_id,
-  JSON_VALUE(event_properties, "$.product_id") AS product_id,
-  JSON_VALUE(event_properties, "$.promotionCode") AS promotion_code,
-  JSON_VALUE(event_properties, "$.payment_provider") AS payment_provider,
-  JSON_VALUE(event_properties, "$.checkout_type") AS checkout_type,
-  JSON_VALUE(event_properties, "$.source_country") AS source_country,
-FROM
-  unioned
+  ERROR(
+    'VIEW BEING DEPRECATION - This view will be completely deleted after 1st of February 2023, please use `fxa_all_events` with filter on `event_category` instead. See DENG-582 for more info.'
```
Comment (relud) [inline sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_stdout_events/view.sql:136]:
```suggestion
    'VIEW DEPRECATED - This view will be completely deleted after 1st of February 2023, please use `fxa_all_events` with filter on `event_category` instead. See DENG-582 for more info.'
```

**Review by relud — APPROVED** [review]
r+wc

**Review by gkabbz — APPROVED** [review]
Not sure exactly why we are doing this but given that relud has approved and I don't want to block the work while I'm on PTO, approving.

**Comment by kik-kik** [comment]
@gkabbz just to provide my rationale or this:
The idea is that the reference will still exist and anything attempting to use this will show the end user a specific error message indicating that this is being replaced by fxa_all_events and a reference to the context around the why. Otherwise, anyone attempting to use this would just see table does not exist error or would continue to create things using this old reference. I felt like this was a nice intermediate step before deleting the views.

**Pull request merged**